integration/spark/src/main/scala/org/apache/carbondata/spark/rdd/CarbonDataRDDFactory.scala

```diff
@@ -775,6 +777,13 @@ object CarbonDataRDDFactory extends Logging {
           loadStatus = CarbonCommonConstants.STORE_LOADSTATUS_FAILURE
           logInfo("DataLoad failure")
           logger.error(ex)
```

Comment:
@Zhangshunyu ....Move logInfo and logger.error below the case match block....also in logInfo log the executorMessage. you can move the variable message also above with default value as "DataLoad failure: " so that the same can be used for logInfo.